test/smoke/README.md

```
@@ -0,0 +1,88 @@
+just test smoke-negative
+```
+
+Subscriber names: `rust`, `python`, `js` (browser), `js-native-node`,
+`js-native-bun`, `c`, `gst`. Publisher names: `rust`, `python`, `js`.
+
+A client whose source build fails fails only its own matrix cells (see
+`mark_broken` in `smoke.sh`); it never aborts the rest of the run.
+
+## Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)